tests/browser/test_em_asm_blocking.cpp

```diff
@@ -7,8 +8,15 @@ std::atomic<int> ret;
 void foo() {
   int len = MAIN_THREAD_EM_ASM_INT({
     var elem = document.getElementById('elem');
+    window.almost_PI = 3.14159;
     return elem.innerText.length;
   });
+  double almost_PI = MAIN_THREAD_EM_ASM_DOUBLE({
+    // read a double from the main thread
+    return window.almost_PI;
+  });
+  printf("almost PI: %f\n", almost_PI);
+  assert(fabs(almost_PI - 3.14159) < 0.001);
```

Comment:
What the changes to the test code?

Reply:
Just some added testing. We tested `MAIN_THREAD_EM_ASM_INT` but not `_DOUBLE`.